src/driver/sqlserver/SqlServerQueryRunner.ts

```diff
@@ -282,28 +282,27 @@ export class SqlServerQueryRunner extends BaseQueryRunner implements QueryRunner
 
         request.query(query);
 
-        // Any event should release the lock.
+        const streamRequest = request.toReadableStream();
```

Comment:
Is this change backward compatible with mssql v6?

Reply:
No, this was introduced in v7. V6 of mssql is quite old and not supported any longer and there have been improvements, including to the streaming implementations.

Reply:
I've updated the PR to target `0.3.0`